templates/sls-app-template-java/src/main/java/com/slf/util/EnvironmentConfig.java

```diff
@@ -29,35 +29,38 @@
 
 	private static Properties props = new Properties();
 	private static String stage = null;
+	private static String functionName = null;
```

Comment:
Why  does functionName need to be static variable?

Reply:
Changed